vispy/visuals/mesh_normals.py

```diff
@@ -15,19 +15,68 @@ class MeshNormalsVisual(LineVisual):
 
     Parameters
     ----------
-    meshdata : instance of MeshData | None
-        The meshdata.
+    meshdata : instance of :class:`~vispy.geometry.meshdata.MeshData`
+        The mesh data.
     primitive : {'face', 'vertex'}
         The primitive type on which to compute and display the normals.
     length : None or float or array-like, optional
-        The length(s) of the normals. If None, the length is set to the median
-        length of the edges of the mesh.
+        The length(s) of the normals. If None, the length is computed with
+        `length_method`.
+    length_method : {'median_edge', 'max_extent'}, default='median_edge'
+        The method to compute the length of the normals (when `length=None`).
+        Methods: 'median_edge', the median edge length; 'max_extent', the
+        maximum side length of the bounding box of the mesh.
+    length_scale : float, default=1.0
+        A scale factor applied to the length computed with `length_method`.
     **kwargs : dict, optional
         Extra arguments to define the appearance of lines. Refer to
-        :class:`~vispy.visuals.line.LineVisual`.
+        :class:`~vispy.visuals.line.line.LineVisual`.
+
+    Examples
+    --------
+    Create a :class:`~vispy.visuals.mesh.MeshVisual` on which to display
+    the normals and get the :class:`~vispy.geometry.meshdata.MeshData`:
+
+    >>> mesh = MeshVisual(vertices=vertices, faces=faces, ...)
+    >>> meshdata = mesh.mesh_data
+
+    Create a visual for the mesh normals:
+
+    >>> normals = MeshNormalsVisual(meshdata)
+    >>> normals.parent = mesh
```

Comment:
I don't think `.parent` is available on the `XVisual` version of the class. Only the `vispy.scene.visuals.MeshNormals` version of the class has that which is added by the `Node` parent class. I could be wrong. Assuming a person puts the same transforms on both visuals, is the `.parent` needed?

Reply:
That's correct, there's no `.parent` here. Thanks.